examples/README.md

```diff
@@ -106,146 +109,139 @@ the wrong directory.
+instead of the default GET. If the call is successful you will get something
+like:
 
     {"version":"2015-12-14T06:44:37.378Z","deploymentId":"e7680e8e-d073-4c57-9f64-e73d8b634398"}
 
-you could check the status again, per the previous section. It should give you `{"message":"App '/hello-world' does not exist"}`
-
+Now you can check the status again, as per the previous section. It should give
+you `{"message":"App '/hello-world' does not exist"}`.
 
-## Where is my Application
 
-In version 0.5.0 the vagrant build does not have traefik in it and so service discovery is a bit convoluted but not too
-bad.
+## Where is my Application?
 
-In a browser, go to the IP of your vagrant cluster. This will be the value in your Vagrantfile as described earlier.
+In a browser, go to the IP of one of your vagrant control nodes. (Some browsers,
+like Firefox, will outright reject the SSL certificate, so you may need to use
+another for this step). This will be the value in your Vagrantfile as described
+earlier.
 
-![mantlui 192.168.242.55/ui](./images/mantlui.png)
+![mantlui 192.168.100.101/ui](./images/mantlui.png)
 
-Choose the Marathon "web UI" button and you should see:
+Choose the Marathon "Web UI" button and you should see something like this:
 
 ![marathonui](./images/marathonui.png)
 
-Click on your application, hello-world (Note: if its not there. you probably deleted it working through the steps above.
-Just start it again. ) and you should see:
+Click on your application, hello-world (if its not there, you probably deleted
+it while working through the steps above, but you can just start it again.)
+and you should see:
 
 ![marathonui at application](./images/marathonapp.png)
 
-You'll notice that there are two instances.  Each one has a line under it in gray `default:<####>` where <####> is some
-port number. In the picture above, ports 9061 and 25312.
+You'll notice that there are two instances. Each one has a line under it in
+gray `default:<####>` where <####> is some port number. In the picture above,
+these are ports 9061 and 25312.
 
-One note.. there are two here becasue the hello-world.json file you submitted asks for two instances to be created.
-This is two seperate hello-world applications.
+There are two instances here becasue the hello-world.json file you submitted
+specifies for two instances to be created. This is two seperate hello-world
+applications, possibly running on different hosts.
 
-If I click on the `default:9061` it will open my browser to default:9061 and get a `webpage is not avaiable` error.
-This is because I don't have default mapped to 192.168.242.55 in my hosts file.  Rather than mess with that.  Lets just
-take the port information and add it to the IP of where we know the vagrant "cluster" is located.   Open the browser to
- 192.168.242.55:9061 (in this example) and we see:
+If you click on the `default:9061` it will open your browser to default:9061 and
```

Comment:
the following section is a bit out of date with the new vagrant setup. `worker-001` instead of `default`, etc.